gen-bundle-image.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+function set_bundle {
+   local version=$1
+   local preset=$2
+   vfkit_bundle=crc_vfkit_${version}_amd64.crcbundle
+   libvirt_bundle=crc_libvirt_${version}_amd64.crcbundle
+   hyperv_bundle=crc_hyperv_${version}_amd64.crcbundle
+
+   if [[ ${preset} = "podman" ]]; then
+       vfkit_bundle=crc_podman_vfkit_${version}_amd64.crcbundle
+       libvirt_bundle=crc_podman_libvirt_${version}_amd64.crcbundle
+       hyperv_bundle=crc_podman_hyperv_${version}_amd64.crcbundle
+   fi
+}
+
+function generate_image {
+   local version=$1
+   local preset=$2
+
+   if [[ ${preset} = "podman" ]]; then
+       cat <<EOF | podman build --os darwin --arch arm64 --tag podman-bundle:darwin-arm64 -f - .
+FROM scratch
+COPY crc_podman_vfkit_${version}_arm64.crcbundle crc_podman_vfkit_${version}_arm64.crcbundle.sig /
+EOF
+   fi
+
+   cat <<EOF | podman build --os darwin --arch amd64 --tag ${preset}-bundle:darwin-amd64 -f - .
+FROM scratch
+COPY ${vfkit_bundle} ${vfkit_bundle}.sig /
+EOF
+
```

Comment:
Not really enthusiastic about copying and pasting this N times. Looks like this could be parametrized relatively easily?
`generate_image "darwin" "amd64" "podman"`? `set_bundle_variables` could probably be removed.

Reply:
@cfergeau you mean to call generate_image for each platform and arch? Also it means we have to create bundle names as per arch and platform so it will be around same so better to have current implementation. `set_bundle_variables` also used in the `sign_bundle_files`. As of now I would differ it.

Reply:
Yes, we'd be creating the bundle names 'manually', just like we already this function already does for the aarch64 bundle: `crc_podman_vfkit_${version}_arm64.crcbundle crc_podman_vfkit_${version}_arm64.crcbundle.sig`. 
If you prefer to keep the copy & pasted Containerfile and `set_bundle_variables`, then `crc_podman_vfkit_${version}_arm64.crcbundle` needs to be moved there for consistency.

Reply:
Added `vfkit_bundle_arm64` for `crc_podman_vfkit_${version}_arm64.crcbundle` for consistency.